### PR TITLE
Vertikal description list 

### PR DESCRIPTION
**Pull request by pethel**
Vi har behov for en vertikal description list. 

image

### CODE REVIEW

File: packages/ffe-lists-react/src/DescriptionList.js [hunk 1]
```diff
@@ -23,6 +24,8 @@ DescriptionList.propTypes = {
     medium: bool,
     /** Large list. Can't be used with `medium` prop */
     large: bool,
+    /** Displays dts and dds inline*/
+    horizontal: bool,
```
Comment (hdaljord) [inline packages/ffe-lists-react/src/DescriptionList.js:28]: Please add this to the index.d.ts file as well (with a ?, since it's optional)

Comment (pethel) [reply packages/ffe-lists-react/src/DescriptionList.js:28]: https://github.com/SpareBank1/designsystem/pull/632/commits/edca781aeb0019122ca5c1bdb53f41a97c0d1c3e

**Review by henrikhermansen — DISMISSED** [review]
At first it seemed strange to make a breaking change to accommodate these needs.

However, on second thought, to not make a breaking change here would require us to add styling which would essentially be reverting the styling we already have. This breaking change makes a lot more sense.

I approve the changes, but it would be great if you could also update the markdown files with some relevant examples, so it will show on design.sparebank1.no.

**Review by henrikhermansen — DISMISSED** [review]

**Review by antidecaf — DISMISSED** [review]

**Review by antidecaf — APPROVED** [review]

**Comment by ivarni** [comment]
Why can't you make the existing view the default so it doesn't have to be a breaking change? As far as I can see there was no reason for this change to be breaking. That creates unnessescary churn in my opinion. 

There's also nothing breaking about the change in the React component. Updating that on its own does not break anything and the changelog should mention that updating the CSS module is required for this new feature.

**Comment by ivarni** [comment]
[peerDependencies](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-lists-react/package.json#L44) should also be updated since the change was made breaking.

**Pull request merged**